src/Language/PureScript/Pretty/JS.hs

```diff
@@ -182,14 +161,13 @@ conditional = mkPattern match
 accessor :: (Emit gen) => Pattern PrinterState JS (gen, JS)
 accessor = mkPattern match
   where
-  match (JSAccessor _ prop val) = Just (emit prop, val)
+  match (JSAccessor _ prop val) = Just (emit $ T.pack $ codePoints prop, val)
```

Comment:
No, I don't think so. I believe `JSAccessor` represents static member access (`a.b`). There's another constructor for computed member access (`a[b]`) called `JSIndexer` or something. Should we make `JSAccessor` accept any property name and automatically render them using computed member access if static member access would be syntactically invalid?